marathon/client.py

```diff
@@ -95,25 +95,23 @@ def _do_request(self, method, path, params=None, data=None):
 
         return response
 
-    def _do_sse_request(self, path, params=None, data=None):
-        from sseclient import SSEClient
-
-        headers = {'Accept': 'text/event-stream'}
-        messages = None
-        servers = list(self.servers)
-        while servers and messages is None:
-            server = servers.pop(0)
+    def _do_sse_request(self, path):
+        for server in list(self.servers):
```

Comment:
Sorry for the delay. I had so solve some personal problems on the last couple of days but I'm still going to fix this !